crates/ruff_linter/src/rules/flake8_bugbear/rules/no_explicit_stacklevel.rs

```diff
@@ -53,8 +63,16 @@ pub(crate) fn no_explicit_stacklevel(checker: &mut Checker, call: &ast::ExprCall
     if call.arguments.find_keyword("stacklevel").is_some() {
         return;
     }
+    let mut diagnostic = Diagnostic::new(NoExplicitStacklevel, call.func.range());
+
+    let edit = add_argument(
+        "stacklevel=2",
```

Comment:
I'm super surprised this method doesn't need to know if this is a keyword or positional argument!

Reply:
I was also surprised... it seems like it would actually be unsafe to use this method for positional arguments, since it always adds the argument to the end? If that's correct and you'd like me to make a followup PR, let me know - I'd be happy to!

Reply:
Yeah feel free to take a look!